CompilerSource/compiler/compile.cpp

```diff
@@ -148,6 +148,109 @@ inline void write_exe_info(const std::string codegen_directory, const EnigmaStru
   wto.close();
 }
 
+#define irrr() if (res) { idpr("Error occurred; see scrollback for details.",-1); return res; }
```

Comment:
Please move these compiler changes to their own PR, if you intend to continue working on those "libassets" files.